### PR TITLE
SettingsHandler: remove some obviously dead code

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: code/src/java/pcgen/core/SettingsHandler.java [hunk 1]
```diff
@@ -1267,6 +1256,14 @@ public static void ensurePathExists(final File path)
 		}
 	}
 
+	public static void setPcgenOutputSheetDir(final File aFile)
```
Comment (thpr) [inline code/src/java/pcgen/core/SettingsHandler.java]: Wait, what?

File: code/src/java/pcgen/core/SettingsHandler.java [hunk 2]
```diff
@@ -806,7 +796,6 @@ public static void getOptionsFromProperties(final PlayerCharacter aPC)
 				.parseInt(dw.toString().substring(0, Math.min(dw.toString().length(), dw.toString().lastIndexOf('.'))));
 			final int height = Integer
 				.parseInt(dh.toString().substring(0, Math.min(dh.toString().length(), dh.toString().lastIndexOf('.'))));
-			d = new Dimension(width, height);
```
Comment (thpr) [inline code/src/java/pcgen/core/SettingsHandler.java]: Wait, you remove Dimension, but not what leads to it?

File: code/src/java/pcgen/core/SettingsHandler.java [hunk 3]
```diff
@@ -2276,19 +2116,6 @@ private static void readFilePaths()
 			in = new FileInputStream(FILE_LOCATION);
 			getFilepathProp().load(in);
 			String fType = SettingsHandler.getFilePaths();
-
-			if ((fType == null) || (fType.length() < 1))
```
Comment (thpr) [inline code/src/java/pcgen/core/SettingsHandler.java]: I'm unsure how this is a safe removal?

File: code/src/java/pcgen/core/SettingsHandler.java (hunk 2 above)
Comment (grimreaper) [reply code/src/java/pcgen/core/SettingsHandler.java]: good call.

File: code/src/java/pcgen/core/SettingsHandler.java (hunk 1 above)
Comment (grimreaper) [reply code/src/java/pcgen/core/SettingsHandler.java]: bad merge conflict resolution

File: code/src/java/pcgen/core/SettingsHandler.java (hunk 3 above)
Comment (grimreaper) [reply code/src/java/pcgen/core/SettingsHandler.java]: I need to go further. Its entirely unused.

**Pull request merged**